src/containers/SignUpForm.tsx

```diff
@@ -27,6 +28,11 @@ const SignUpForm = () => {
         Create encrypted, resilient, and cross-application data storage in
         seconds.
       </p>
+      <MessageBox
+        type="error"
+        message="The email is already registered"
+        title="Error"
+      />
```

Comment:
This is the new Component to display errors, styles for warning (yellow) to be added...